notebooks/EVA-release-RSIDquery.ipynb

```diff
@@ -0,0 +1,179 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(output)"
```

Comment:
I think this output is too large and difficult to read. I would limit to only show detail of the first submitted variant. 
```suggestion
    "print(json.dumps(output[0], indent=2))"
```
You could show the number of variant with something like 
```
print('Number of submitted variant=' + str(len(output)))
```